examples/GPS_HardwareSerial_EchoTest/GPS_HardwareSerial_EchoTest.ino

```diff
@@ -41,3 +44,8 @@ void loop() {
     Serial.write(c);
   }
 }
+
+#else // Do nothing for boards without hardware serial
```

Comment:
i think we could print an #error but doesnt make sense to do nothing - they'll think its just not working :)